### PR TITLE
feat: leaderboard page added

### PR DESCRIPTION
**Pull request by shobit1337**
- leader board page added 
Demo:
image

### CODE REVIEW

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: css/styles.css [hunk 1]
```diff
@@ -98,20 +98,32 @@
   flex-basis: 45%;
 }
 
+.create-btn {
+  background-color: rgb(242, 242, 242);
+}
+.create-btn:hover {
+  background-color: rgb(242, 242, 242);
+}
```
Comment (Tarandeep-s1ngh) [inline css/styles.css]: Suggestion: You can use variables for these colors

File: pages/leaderboard.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cloud-ui-css.netlify.app/css/main.css"
+    />
+    <link rel="stylesheet" href="/css/styles.css" />
+    <title>Quizmill</title>
```
Comment (Tarandeep-s1ngh) [inline pages/leaderboard.html]: Update the title for this page

**Review by simran61 — APPROVED** [review]

File: pages/leaderboard.html (hunk 2 above)
Comment (shobit1337) [reply pages/leaderboard.html]: Done

File: css/styles.css (hunk 1 above)
Comment (shobit1337) [reply css/styles.css]: Done, Thanks for pointing it out @Tarandeep-s1ngh

**Pull request merged**